src/state/audit/saga.js

```diff
@@ -48,7 +50,10 @@ function* fetchPositionsAudit({ payload: ids }) {
     }
     const auth = yield select(selectAuth)
     const query = yield select(getQuery)
-    const { result = [], error } = yield call(getReqPositionsAudit, auth, query, targetIds, 0)
+    const { result: resulto, error: erroro } = yield call(getReqPositionsAudit, 0, auth, query, targetIds)
+    const { result = {}, error } = yield call(
+      fetchNext, resulto, erroro, getReqPositionsAudit, 0, auth, query, targetIds,
+    )
```

Comment:
@gasolin Might it be cleaner to pass a single object instead of 8 or 9 parameters to one function?    That way the order of the parameters becomes unimportant and it is easy to know what every argument is.   thanks.

Reply:
resolved and updated related tests. Thanks